.circleci/config.yml

```diff
@@ -1,93 +1,129 @@
+version: 2.1
+executors:
+  default:
+    docker:
+      - image: cimg/node:17.3
```

Comment:
should use node 18 if possible. The even numbers of node will have LTS

https://nodesource.com/blog/understanding-how-node-js-release-lines-work/